plugins/CMakeLists.txt

```diff
@@ -175,8 +189,11 @@ if(BUILD_SUPPORTED)
     add_subdirectory(tweak)
     dfhack_plugin(workflow workflow.cpp LINK_LIBRARIES lua)
     dfhack_plugin(workNow workNow.cpp)
+    dfhack_plugin(xlsxreader xlsxreader.cpp LINK_LIBRARIES lua xlsxio_read zip expat ${LIB_Z_LIB})
     dfhack_plugin(zone zone.cpp LINK_LIBRARIES lua)
 endif()
+target_include_directories(xlsxreader PRIVATE ${XLSXIO_INSTALL_DIR}/include)
+add_dependencies(xlsxreader xlsxio_project)
```

Comment:
A bit late here, but this should have gone in the `if (BUILD_SUPPORTED)` block above as well. For instance, df-ai builds with BUILD_SUPPORTED=OFF, and this fails because the `xlsxreader` target isn't created: https://buildmaster.lubar.me/executions/execution-details?executionId=504238. I'll push a fix.

Reply:
Ack. Thanks!